internal/store/postgres/job/job_repository.go

```diff
@@ -699,6 +699,13 @@ VALUES (
 				upstream.UpstreamType, upstream.UpstreamState)
 		}
 
+		if errors.IsPgErrorCode(err, errors.ErrPgCodeUniqueConstraints) {
```

Comment:
Let's move it to line after `if err != nil`

Reply:
if we move it after that line, i dont think this function will be called. Instead maybe i should add `if err != nil` in the implementation wdyt @irainia

Reply:
I mean let's put it inside the `if err != nil` statement, a few lines below this line

Reply:
sure